webapp/components/profile_popover/profile_popover.jsx

```diff
@@ -243,16 +260,31 @@ export default class ProfilePopover extends React.Component {
                     </a>
                 </div>
             );
-            dataContent.push(webrtc);
         }
 
+        return '';
+    }
+
+    render() {
+        const popoverProps = Object.assign({}, this.props);
```

Comment:
there are some elements needed by the `<Popover />`. I just changed it so it's explicitly indicated:
```
<Popover
    arrowOffsetLeft={this.props.arrowOffsetLeft}
    arrowOffsetTop={this.props.arrowOffsetTop}
    positionLeft={this.props.positionLeft}
    positionTop={this.props.positionTop}
```